pkg/apis/storage/validation/validation.go

```diff
@@ -174,11 +171,7 @@ func validateVolumeAttachmentSource(source *storage.VolumeAttachmentSource, fldP
 	allErrs := field.ErrorList{}
 	switch {
 	case source.InlineVolumeSpec == nil && source.PersistentVolumeName == nil:
-		if utilfeature.DefaultFeatureGate.Enabled(features.CSIMigration) {
```

Comment:
restore this, it only affects the validation message, not whether  the object is considered valid or not (both branches lead to an error)

Reply:
@liggitt 
Your comment has been addressed